src/pages/journal/components/test-outcome/test-outcome.ts

```diff
@@ -106,6 +111,13 @@ export class TestOutcomeComponent {
     this.modal.present();
   }
 
+  displayForceCheckModal = (): void => {
+    const options = { cssClass: 'mes-modal-alert text-zoom-regular' };
+    this.modal = this.modalController.create('JournalForceCheckModal', {}, options);
```

Comment:
As this is a new page it needs to be added to page-names.constants.ts and then we should use that reference instead of a string here